src/Tizen.NUI/src/public/Window/WindowEvent.cs

```diff
@@ -539,6 +542,34 @@ public partial class Window
             }
         }
 
+        /// <summary>
+        /// DeviceInfoEvent event.
```

Comment:
Could you add more description? It's hard to guess when this event is triggered.